spec/index.html

```diff
@@ -663,13 +663,13 @@ <h2>Changes</h2>
       The document title changed from "SPARQL Query Results XML Format (Second Edition)" to "SPARQL 1.2 Query Results XML Format".
     </section>
 
-    <section id="mime">
+    <section id="mediatype">
```

Comment:
Thse id changes might break some links to this spec

Reply:
Someone recently was making similar changes (I don't remember if it was within this group or not), and solved that potential issue by adding something like `<div id="mime"></div>` immediately following the `<section id="mediatype">`. (The other person's addition did something clever so it was clear that this new `div` was just to maintain the old fragid.) This lets old links to `#mime` still get to the right place in the revised document, and (hopefully) new links will be to the new `#mediatype`.

Reply:
```suggestion
    <section id="mediatype"><div id="mime"></div>
```